tests/performance/common/common.go

```diff
@@ -113,7 +113,7 @@ func HandleEdgeDeployment(cloudhub, depHandler, nodeHandler, cmHandler, imgURL,
 			}
 		}
 		return count
-	}, "60s", "2s").Should(Equal(numOfNodes), "Nodes register to the k8s master is unsuccessfull !!")
+	}, "1200s", "2s").Should(Equal(numOfNodes), "Nodes register to the k8s master is unsuccessfull !!")
```

Comment:
1200s = 20mins, The test will be stuck for 20 mins if node registration has failed. can we keep short time duration ?

Reply:
@sids-b I tested about 50 edge nodes. it will take more than 240s.
if we test 500 edge nodes, i guess 1200s is better so that it will not be timeout:(

Reply:
OK, Thanks :)